index.js

```diff
@@ -63,11 +67,18 @@ Nanocomponent.prototype.render = function () {
   }
 }
 
+Nanocomponent.prototype.rerender = function () {
+  assert(this.element, 'nanocomponent: cant rerender on an unmounted dom node')
```

Comment:
I think the only thing left is to turn this into a NOOP return that warns with the same message.